es-core/src/components/VideoComponent.cpp

```diff
@@ -221,6 +212,15 @@ void VideoComponent::applyTheme(const std::shared_ptr<ThemeData>& theme, const s
 	if (elem->has("showSnapshotDelay"))
 		mConfig.showSnapshotDelay = elem->get<bool>("showSnapshotDelay");
 
+	setRotation(0.0);
```

Comment:
@pjft Yes we are saying the same thing. I was just giving a use case where rotation+resize might be used hundreds of times a second and it would be quite unnecessary to keep on setting it to zero. If a dev is unfamiliar with this, debugging would waste time.